engine/gamesys/src/gamesys/test/collision_object/velocity_threshold.script

```diff
@@ -27,29 +27,30 @@ local counter = 0
 
 
 function update(self, dt)
-  counter = counter + 1    
+  counter = counter + 1
   local velocity1 = math.floor(vmath.length(go.get("body1-go#body-co", "linear_velocity")))
   local velocity2 = math.floor(vmath.length(go.get("body2-go#body-co", "linear_velocity")))
-  --print("Frame", counter,":", velocity1, "-", velocity2)
-  
+  -- print("Frame", counter,":", velocity1, "-", velocity2)
+
   -- object hit at frames 16. At 10, object 2 should stand still
+  -- box2d v3: This does not seem to be the case. When the objects are spawned, object 2 has a velocity from the get-go.
```

Comment:
Does this affect how current games work?
If so, perhaps we need to collect these "nuggets" in the release notes?